One solution proposed on the ports mailing list was to uninstall the ImageMagick port, rebuild editors/koffice-kde3 and then reinstall ImageMagick, while this works it is unacceptable to have to uninstall a port to get another port to build.

The correct solution is to patch the source configure script to skip the detection of ImageMagick when GraphicsMagick is installed.  see attached patch:

Patch attached with submission follows:

--- configure.orig	2007-05-30 16:56:35.000000000 -0500
+++ configure	2008-07-10 05:51:45.898085206 -0500
@@ -33900,8 +33900,10 @@
 
 # Check for ImageMagick...
 
 have_imagemagick=no
 
+if test "$have_graphicsmagick" = "no"; then
+
    { echo "$as_me:$LINENO: checking for Magick-config" >&5
 echo $ECHO_N "checking for Magick-config... $ECHO_C" >&6; }
    if test -n "$MAGICK_CONFIG"; then
@@ -34014,7 +34017,7 @@
     LIBMAGICK_RPATH=
 fi
 
-
+fi
 
 
 


